### PR TITLE
Make wtest more robust

### PR DESCRIPTION
**Pull request by totto82**
We only want to open wells that can operate under both economical and physical reasons.

Depens on OPM/opm-common#2711

### CODE REVIEW

File: opm/simulators/wells/WellInterface_impl.hpp [hunk 1]
```diff
@@ -261,13 +238,27 @@ namespace Opm
...
+            bool converged = solveWellForTesting(simulator, well_state_copy, group_state, deferred_logger);
+            if (!converged) {
+                const std::string msg = " well " + this->name() + " did not get converged during well testing";
+                deferred_logger.debug(msg);
+                return;
+            }
+
+            updateWellOperability(simulator, well_state_copy, deferred_logger);
+            if ( !this->isOperable() ) {
+                const std::string msg = " well " + this->name() + " is not operable during well testing for physical reason";
```
Comment (joakim-hove) [inline opm/simulators/wells/WellInterface_impl.hpp:250]:
```C++
auto msg = fmt::format("Well {} is not operable during well testing for physical reason", this->name());
```

**Comment by joakim-hove** [comment]
Upstream in opm-common is ready to be merged.

File: opm/simulators/wells/WellInterface_impl.hpp (hunk 1 above)
Comment (bska) [reply opm/simulators/wells/WellInterface_impl.hpp:250]: If we do update this, can we rephrase the message to be less ambiguous too?  E.g, something along the lines of

> WTEST: Well &lt;name&gt; is not operable (physical reasons)

**Comment by bska** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks.  I'll merge into master.

**Pull request merged**